README.md

```diff
@@ -212,6 +213,14 @@ Combines each element with each element of a given other array.
 // => [(1, "A"), (1, "B"), (2, "A"), (2, "B"), (3, "A"), (3, "B")]
 ```
 
+#### .distinct()
```

Comment:
I'm not sure about the naming, I'm used to something like `uniq()` from Ruby. But `distinct()` would be okay too. The bigger concern from my point of view is: Why not use a `Set` istead if I want uniq objects? I mean, that's what it was made for. And if you need an ordered set, I'd rather introduce a `OrderedSet` type instead of adding a `.distinct()` method to Swift. But that could be done in a different PR.